column/api/common/utils.py

```diff
@@ -0,0 +1,7 @@
+import cerberus
```

Comment:
Both two are popular now. 
From my perspective, they are similar. Both ways are good.

Do you have any preference which way to go?